app/jobs/catalogue_check_job.rb

```diff
@@ -0,0 +1,76 @@
+class CatalogueCheckWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'

```

Comment:
I've made capybara and poltergeist gem available in all environment in PR #5. However, i found that the code is not working without requires on line 4 and line 5.